docs/resources/table.md

```diff
@@ -111,7 +111,7 @@ resource "snowflake_table" "table" {
 Required:
 
 - `name` (String) Column name
-- `type` (String) Column type, e.g. VARIANT
+- `type` (String) Column type, e.g. VARIANT. For a full list of column types, see [Summary of Data Types](https://docs.snowflake.com/en/sql-reference/intro-summary-data-types).
```

Comment:
This file is auto-generated, so it should not be updated directly.

You can instead add it to `pkg/resources/table.go` and run `make docs`; that will update this file.